### ISSUE TITLE
New git tag/release with LLVM 11 support

### ISSUE THREAD

**Issue by isuruf** [jrprice/Oclgrind#196]
It would be good to have a new release so that I can package it in conda.

**Comment by jrprice** [comment]
Yes. I'm currently planning to do the next release soon after that LLVM 12 release so that it will support both 11 and 12, which will give it a longer shelf life.
